bloom/git.py

```diff
@@ -686,7 +687,9 @@ def get_last_tag_by_version(directory=None):
     versions = []
     for line in output.splitlines():
         tags.append(line.strip())
-        versions.append(parse_version(line.strip()))
+        ver = re.match("[0-9]+.[0-9]+.[0-9]+", line)
```

Comment:
```suggestion
        ver = re.match(r"[0-9]+\.[0-9]+\.[0-9]+", line)
```

Since dots are regex metacharacters they need to be escaped for an exact match. Or is the intent that any character could separate these digits?

Reply:
This is likely what was intended.

Reply:
Resolved in cd2031799a3e00296d70a47f0440326c67ea64d6